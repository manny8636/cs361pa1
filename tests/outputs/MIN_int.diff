--- outputs/MIN_int.txt	2022-02-13 13:37:47.303555326 -0500
+++ expected/MIN_int.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,2 @@
+VALUE [integer]: 123
 Success!
