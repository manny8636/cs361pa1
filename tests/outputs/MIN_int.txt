Parsing ../data/int.txt failed
