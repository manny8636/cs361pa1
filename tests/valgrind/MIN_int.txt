==372105== Memcheck, a memory error detector
==372105== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372105== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372105== Command: ../json -v ../data/int.txt
==372105== 
==372105== 
==372105== HEAP SUMMARY:
==372105==     in use at exit: 2,064 bytes in 1 blocks
==372105==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372105== 
==372105== LEAK SUMMARY:
==372105==    definitely lost: 2,064 bytes in 1 blocks
==372105==    indirectly lost: 0 bytes in 0 blocks
==372105==      possibly lost: 0 bytes in 0 blocks
==372105==    still reachable: 0 bytes in 0 blocks
==372105==         suppressed: 0 bytes in 0 blocks
==372105== Rerun with --leak-check=full to see details of leaked memory
==372105== 
==372105== For lists of detected and suppressed errors, rerun with: -s
==372105== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
