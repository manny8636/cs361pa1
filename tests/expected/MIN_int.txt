VALUE [integer]: 123
Success!
